src/pages/workspaces/workspace/analysis/modals/AnalysisModal.test.ts

```diff
@@ -84,10 +82,12 @@ describe('AnalysisModal', () => {
   beforeEach(() => {
     // Arrange
     asMockedFn(useAnalysisFiles).mockImplementation(() => ({
-      refresh: () => Promise.resolve(),
+      refreshFileStore: () => Promise.resolve(),
       loadedState: { state: [], status: 'Ready' },
       create: createFunc,
-      pendingCreate: { status: 'Ready', state: true }
+      pendingCreate: { status: 'Ready', state: true },
```

Comment:
There might be a misunderstanding of the intitial state on these... Both are using useLoadedData under the hood... which means the initial state is actually `{ status: 'None' }`